src/mailchimp/mailchimp_manager.py

```diff
@@ -60,8 +72,14 @@ def ping(self) -> Response:
         Raises:
             TODO: Detail potential errors
         """
-        raise NotImplementedError
-    
+        try:
+                client = MailchimpMarketing.Client()
```

Comment:
The client should just be a protected attribute of the mailchimp manager that is set in the `set_authorisation` method. That way, you don't have to recreate the client object every time `ping()` is called